Makefile

```diff
@@ -81,13 +81,6 @@ clean:
 	cargo clean
 	go clean ./...
 
-# DEPRECATED - please use build-stellar-rpc instead
-# the build-stellar-rpc build target is an optimized build target used by
-# https://github.com/stellar/pipelines/blob/master/stellar-rpc/Jenkinsfile-stellar-rpc-package-builder
-# as part of the package building.
-build-stellar-rpc: build-libs
```

Comment:
Ahhh, this surely happened through an automatic rename from soroban-rpc to stellar-rpc